docs/README.md

```diff
@@ -101,71 +117,68 @@ module.exports = {
+- Fields are added to the module with the new `fields` syntax. New fields are added via the `add` property.
+- You can also `remove` fields by passing an array for `remove`.
+- For convenience, fields are now configured as an object, rather than an array.
 
-* Every area must be configured as part of the schema for a page type or piece type.
+### Templates
 
-* Fields are added to the module with the new `fields` feature. New fields go in the `add` property. You can also `remove` fields by passing an array for `remove`. For convenience, fields are now configured as an object, rather than an array.
+The following is our template for the homepage. There are a few important changes to templates in Apostrophe 3, but the syntax should be familiar.
 
-### The template for the home page
+- Areas are added to the template with the new `area` nunjucks tag. There is no `apos.area` helper function anymore.
+- You don't configure the area here. You do that in the `index.js` file for the page type or piece type. In the template you just pass the page and the name of the area.
 
-Check out `modules/@apostrophecms/home-page/views/page.html`:
+```jsx
+// modules/@apostrophecms/home-page/views/page.html
 
-```
 {% extends "layout.html" %}
 
 {% block main %}
   {% area data.page, 'main' %}
 {% endblock %}
-```
 
-**"What's changed from A2 in this code?"**
-
-1. Areas are added to the template with the new `area` nunjucks tag. There is no `apos.area` helper function anymore.
-
-2. You don't configure the area here. You do that in the `index.js` file for the page type or piece type. In the template you just pass the page and the name of the area.
+```
 
-### Adding more standard widgets
+### Widgets
 
 Try adding more sub-properties to `widgets` in `index.js`:
 
-```javascript
-'@apostrophecms/video': {},
-'@apostrophecms/raw-html': {},
-'@apostrophecms/image': {}
+```jsx
+// modules/@apostrophecms/home-page/index.js
+
+...
+widgets: {
+	'@apostrophecms/video': {},
+	'@apostrophecms/raw-html': {},
+	'@apostrophecms/image': {}
+...
 ```
 
-You will also need to configure a CSS class for the image widget, because A3 does not impose strong opinions on your front end code. First make a folder for project-level configuration of the image-widget module:
+**Configuring a Widget**
+
+Apostrophe 3 does not impose any front-end opinions regarding widgets, and thus its necessary to configure them properly with CSS classes for styling. In this example, we'll configure our image-widget to have a class. Start by creating a directory for project-level configuration. Using your terminal:
 
 ```
 mkdir modules/@apostrophecms/image-widget
 ```
 
-Then you can configure it:
+Then, you can configure it by creating an `index.js` file in that directory.
 
-```
-// in modules/@apostrophecms/image-widget/index.js
+```jsx
+// modules/@apostrophecms/image-widget/index.js
 module.exports = {
   options: {
-    className: 'a3-widget-image'
+    className: 'home-image'
```

Comment:
This is not specific to the home page so definitely not `home-`. The a3 prefix is a nice convention to separate it from custom widgets in the project, but it's not essential.

Reply:
Point taken, but we shouldn't recommend using Apostrophe as a reference when making classes....how about 
`full-width-image`, since that's what max-width 100% does.